karapace/schema_registry_apis.py

```diff
@@ -44,6 +45,10 @@ class SchemaErrorCodes(Enum):
     INVALID_VERSION_ID = 42202
     INVALID_COMPATIBILITY_LEVEL = 42203
     INVALID_AVRO_SCHEMA = 44201
+    INVALID_PROTOBUF_SCHEMA = 44202
+    INVALID_REFERECES = 44203
+    REFERENCES_SUPPORT_NOT_IMPLEMENTED = 44501
+    SCHEMAVERSION_HAS_REFERENCES = 44503
```

Comment:
I would reserve error codes 44201 to 442xx for Avro, Protobuf, JSONSchema and future formats.

Nit: typo in `INVALID_REFERECES`.

Maybe following? 
```suggestion
    INVALID_REFERENCES = 44301
    REFERENCES_SUPPORT_NOT_IMPLEMENTED = 44302
    SCHEMAVERSION_HAS_REFERENCES = 44303
```